### PR TITLE
building doxygen is a prereq on linux

### PR DESCRIPTION
**Pull request by arkadianriver**
I was unable to run `make` without running `make build_doxygen_adoc` first.
Making this PR from Write the Docs after troubleshooting the issue with Nellie.

### CODE REVIEW

**Comment by arkadianriver** [comment]
Note that the `gcc-arm-node-eabi` dependency for the doxygen build was true for me on Pop_OS 22.04. Someone on Ubuntu 18 was unable to install this and didn't need to.

File: README.md [hunk 1]
```diff
@@ -128,29 +135,13 @@ $ make serve_html
...
-
-```
-$ make build_doxygen_doc
-$ make
-```
-
-and clean up afterwards by using,
+### Cleaning up afterward
 
+You can revert your repositoryt to a pristine state by running,
```
Comment (cmangla) [inline README.md:140]: Typo `repositoryt`

Comment (arkadianriver) [reply README.md]: 🤦 Thank you!

**Comment by aallan** [comment]
The Linux instructions are intended for Raspberry Pi OS, although they should be valid for most Debian-derived distributions. There are so many distributions that we can't provide instructions for all of them. I'll double-check next week when I get back home from WTD as to whether the instructions as written work on the current version of Raspberry Pi OS.

**Comment by lurch** [comment]
See #2931

**Pull request closed without merging**